.release-notes/4147.md

```diff
@@ -0,0 +1,12 @@
+## change error message
+error message in the situation:
+```
+actor Main
+        let my_test_class: TestClass= TestClass
+        new create(env: Env) =>
+                """placeholder"""
+        fun test() =>
+                my_test_class.x= 1
+```
+was changed from ```not safe to write right side to left side``` to 
+```cannot write to a field in a box function. If you are trying to change state in a function use fun ref.```
```

Comment:
as written this shouldnt be two lines. also,

``` is for multiline code blocks
` is for inline ones.

I think you might want to use "" instead of ``, but that is up to you.